modules/virt-creating-a-service-backed-by-virtual-machines.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-creating-a-service-backed-by-virtual-machines_{context}"]
+
+= Creating a service backed by virtual machines
+
+To create a service backed by a virtual machine, you start the `VirtualMachineInstance`,
```

Comment:
Shouldn't it be `VirtualMachine` instead of `VirtualMachineInstance`? VMI is started immediately after creation and user doesn't have control of it afaik, so the sentence doesn't make sense to me.

Reply:
Changed VirtualMachineInstance to VirtualMachine.